buildSrc/src/main/kotlin/Config.kt

```diff
@@ -0,0 +1,35 @@
+import org.gradle.kotlin.dsl.extra
+
+object Artifact {
+  const val groupdId = "com.mercari.remotedata"
+  const val version = "1.0.0"
+}
+
+object MavenUrl {
+  const val spekDev = "https://dl.bintray.com/spekframework/spek-dev"
+}
+
+object Version {
+  const val bintray = "1.8.4"
+  const val kotlin = "1.3.10"
+  const val kluent = "1.49"
+  const val spek = "2.0.1"
+  const val junitPlatform = "1.2.0"
+  const val jacoco = "0.8.4"
```

Comment:
[should] This `const val` is not used. please use or remove it.

Reply:
I forgot to use that 🙇